api/src/controllers/object.controller.ts

```diff
@@ -175,6 +175,41 @@ class ObjectController {
       status: 'success',
     });
   }
+
+  public async increaseQuantity(request: Request, response: Response): Promise<void> {
+    const { apitoken } = request.headers;
+    const session = this.memberService.decodeMemberToken(<string> apitoken);
+    if(!session) {
+      response.status(400).json({
+        error: 'Invalid or missing token.',
+      });
+      return;
+    }
+    try{
+      const object = await this.objectService.findById(request.body.objectId);
+      const increase = request.body.qty;
+      const member = await this.memberService.find({ id: session.id });
+      const wallet = await this.walletService.findById(member.wallet_id);
+      const sellerFee = this.objectService.getSellerFee(increase, object.price);
+      if (object.member_id !== session.id) {
+        throw new Error('You do not own this object!');
+      }
+      if (sellerFee > wallet.balance) {
+        throw new Error('You do not have enough CCs.');
+      }
+      if (object.limit && object.quantity + increase > object.limit){
```

Comment:
This was done this way because it was common for people to upload a small portion of a limited object at first and then add the remaining afterwards. For example, only uploading 20 of a 1/25 to guarantee the creator can get some examples for themselves if they couldn't make the drop.